website/docs/developer/research/research-localstack.md

```diff
@@ -183,24 +197,27 @@ Some examples of CLI commands and their corresponding outputs are shown below.
             "tags": {},
             "disableExecuteApiEndpoint": false
         }
-        
-  `PARENT_RESOURCE_ID=$(awslocal apigateway get-resources --rest-api-id 0zqyew47ie --query 'items[?path==`/`].id' --output text)`
-  
-  `awslocal apigateway create-resource --rest-api-id 0zqyew47ie --parent-id  $PARENT_RESOURCE_ID --path-part "{test}"`
+```
+  ```commandline
+  PARENT_RESOURCE_ID=$(awslocal apigateway get-resources --rest-api-id 0zqyew47ie --query 'items[?path==`/`].id' --output text)
 
+  awslocal apigateway create-resource --rest-api-id 0zqyew47ie --parent-id  $PARENT_RESOURCE_ID --path-part "{test}
+  ```
+  ```commandline       
      Response: 
     {
         "id": "7z17qim7fw",
         "parentId": "d22pls3twh",
         "pathPart": "{test}",
         "path": "/{test}"
 }
+  ```
         
-        
-
 #### Lambda function
-`awslocal lambda create-function --function-name test-function --runtime python3.7 --zip-file fileb://test.zip --handler test.lambda_handler --role test`
-
+  ```commandline
```

Comment:
See notes above about combining the code blocks and adding some comments for users.